src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -244,6 +256,19 @@ export class OperationDetails {
         this.working(false);
     }
 
+    public async loadExamples(operation: Operation): Promise<void> {
+        const representations = operation.request.meaningfulRepresentations().map(a => {return {...a}});
+        let requestExamples = {};
+        if (representations && representations.length) {
```

Comment:
instead of if and for, you could use something like
`representations?.foreach((representation, index) => {...}`

If you want to go with the for approach, I think it would be better to use `let` instead of `var` here. 
The variables that are defined with `let` statement have block scope.
The variables that are defined with `var` statement have function scope.
some reference: https://levelup.gitconnected.com/const-vs-let-vs-var-in-javascript-which-one-should-you-use-c56cf9b9e2a3

Reply:
updated thank you